modules/persistent-storage-csi-migration-vsphere-to-4-14.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-vsphere-to-4.14_{context}"]
+= Updating from {product-title} 4.12 to 4.14 with vSphere in-tree PVs
+
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.12 to 4.14, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} updates are blocked. After updating vSphere, your {product-title} update can occur and automatic CSI migration for vSphere occurs automatically by default.
+
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} update by running the following commands to perform an administrator acknowledgment:
+
+[source, terminal]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.12-kube-1.26-vsphere-migration-in-4.14":"true"}}' --type=merge
+----
+
+[source, terminal]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.13-kube-1.27-vsphere-migration-in-4.1":"true"}}' –type=merge
```

Comment:
typo in the key
ack-4.13-kube-1.27-vsphere-migration-in-4.1 -> ack-4.13-kube-1.27-vsphere-migration-in-4.14

Reply:
Good catch. Fixed.